<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt Paint</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="toolBar" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnDrawLine">
         <property name="text">
          <string>Draw Line</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDrawCircle">
         <property name="text">
          <string>Draw Circle</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDrawPolygon">
         <property name="text">
          <string>Draw Polygon</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDrawRectangle">
         <property name="text">
          <string>Draw Rectangle</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnClip">
         <property name="text">
          <string>Clip</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnFill">
         <property name="text">
          <string>Fill</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnImageFill">
         <property name="text">
          <string>Fill Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnFloodFill">
         <property name="text">
          <string>Flood Fill</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnResetMode">
         <property name="text">
          <string>Reset Mode</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnChangeColor">
         <property name="text">
          <string>Change Color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnThicken">
         <property name="text">
          <string>Thicken</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnToggleAntiAliasing">
         <property name="text">
          <string>Toggle Anti-aliasing</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnSave">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLoad">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRemoveAll">
         <property name="text">
          <string>Remove All</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
